From 89c705f70b8be3dfd778ea2f778470aabd871798 Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Thu, 9 Aug 2018 15:20:15 +0000
Subject: [PATCH] MANTA-1387 libmanta should log and ignore BucketVersionError
 Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com> Approved by:
 Kelly McLaughlin <kelly.mclaughlin@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index ad7051f..e2027f8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "libmanta",
     "description": "common functions et al for manta services",
-    "version": "1.0.0",
+    "version": "1.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "main": "./lib/index.js",
-- 
2.21.0

